src/resources/resources.service.ts

```diff
@@ -0,0 +1,98 @@
+import { BadRequestException, Injectable, UnauthorizedException } from '@nestjs/common';
+import { CreateResourceDto } from './dto/create-resource.dto';
+import { UpdateResourceDto } from './dto/update-resource.dto';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { TeamResource } from '@prisma/client';
+
+const userId = 6;
```

Comment:
how come userId is hardcoded? for now, it can come from request body till we have auth sort which will pass the userId

I believe this userId should also be a UUID, not int

there's a difference between userId, and voyageTeamMemberId. UserId is like a Chingu member user id, while voyageTeamMemberId is their IDs in voyages. It is set up this way because a user can participate in multiple voyages

Reply:
other reasons you can't hardcode here is the endpoint will basically be usable by frontend, and Ids can change everytime the data is reseeded

Reply:
I guess just a bad habit from using workarounds like that before while solo coding, it makes complete sense to me now why that's not workable.

Also completely clear to me now the difference between the voyageTeamMemberId and userId.